EDITORS.rst

```diff
@@ -0,0 +1,35 @@
+Welcome
+=======
+
+Thank you for helping edit the NixOS "Weekly" Newsletter!
+
+Creating a new post
+===================
+
+
+1. Copy `template.rst <./template.rst>`_ to ``content/<year>/<number>-<title>.rst``
+
+2. Make changes
+
+   Running ``nix-shell --command "pelican -r"`` will rebuild the issue
+   whenever new content is discovered. Check the ``drafts`` folder for
+   the output.
+
+   Alternatively, ``nix-build`` will build and produce ``result``.
+
+3. Create a pull-request against ``master``.
+
+   You'll be able to view the staging site at http://weekly.nixos.org/preview.
+
+4. Ask for contributions.
+
+   Some good places to do so may include:
+
+   - the mailing list
+   - social media
+   - friends/coworkers
```

Comment:
I would leave this out. sure you can ask you friends/coworkes, but i think we want to document here the minimal things an editor should do, otherwise the document will be too big with things that you *might* do.